### PR TITLE
ibazel 0.21.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/ibazel.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.20.0.tar.gz"
+  sha256 "7b0f98006b32d2ad5eee38eb9b363005968abf88d235e31fc6dd2a8c4336f33d"
+  license "Apache-2.0"
+
+  depends_on "bazel" => [:build, :test]
+  depends_on "go" => [:build, :test]
+  depends_on xcode: :build
+
+  # Patch to use Homebrew's Go
+  patch :DATA
```
Comment (chenrui333) [inline Formula/ibazel.rb:13]: This would be permanent patch to make the build work with homebrew's go sdk

Comment (SMillerDev) [reply Formula/ibazel.rb:13]: It would need to be upstreamed nonetheless. If it can't work with Homebrew dependencies it should not be in core

Comment (chenrui333) [reply Formula/ibazel.rb:13]: - actually this is addressed by rules_go now, https://github.com/bazelbuild/rules_go/issues/3302 (https://github.com/bazelbuild/rules_go/pull/3303)

File: Formula/ibazel.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.20.0.tar.gz"
+  sha256 "7b0f98006b32d2ad5eee38eb9b363005968abf88d235e31fc6dd2a8c4336f33d"
+  license "Apache-2.0"
+
+  depends_on "bazel" => [:build, :test]
+  depends_on "go" => [:build, :test]
+  depends_on :macos
+
+  def install
+    inreplace ".bazelversion", "5.3.1", "5.3.2"
```
Comment (SMillerDev) [inline Formula/ibazel.rb:13]: Why?

Comment (chenrui333) [reply Formula/ibazel.rb:13]: there is some build issue

```
ERROR: The project you're trying to build requires Bazel 5.3.1 (specified in /private/tmp/ibazel-20221117-33447-o7pxmb/bazel-watcher-0.20.0/.bazelversion), but it wasn't found in /opt/homebrew/Cellar/bazel/5.3.2/libexec/bin.
```

Should be covered by https://github.com/bazelbuild/bazel-watcher/pull/556

Comment (SMillerDev) [reply Formula/ibazel.rb:15]: We should just wait for the next release then

Comment (chenrui333) [reply Formula/ibazel.rb:15]: yeah, that works for me.

**Comment by chenrui333** [comment]
@SMillerDev should be good now.

**Pull request closed without merging**